src/parsers/power_models_data.jl

```diff
@@ -808,6 +873,33 @@ function read_branch!(
     end
 end
 
+function read_3w_transformer!(
+    sys::System,
+    data::Dict,
+    bus_number_to_bus::Dict{Int, ACBus};
+    kwargs...,
+)
+    @info "Reading 3W transformer data"
+    if !haskey(data, "3w_transformer")
+        @info "There is no 3W transformer data in this file"
+        return
+    end
+    _get_name = get(kwargs, :xfrm_3w_name_formatter, _get_pm_3w_name)
```

Comment:
Shouldn't the default be to use the `name` field?

Reply:
No, for branches we usually want the name to use like the buses that connect + name.